src/components/iap/stripe-plan-choices.json

```diff
@@ -0,0 +1,14 @@
+{
+  "collectionName": "components_iap_stripe_plan_choices",
+  "info": {
+    "displayName": "stripePlanChoices",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "stripePlanChoices": {
+      "type": "string",
```

Comment:
Missing `minLength` for validation?

Reply:
Added a regex checking that the strings start with price_ or plan_ instead of having a minimum length.